src/test/java/com/cloudbees/plugins/credentials/CredentialsProviderTest.java

```diff
@@ -457,4 +458,14 @@ public void credentialsSortedByNameInUI() {
         assertThat(options.get(0).value, is("2"));
         assertThat(options.get(1).value, is("1"));
     }
+
+    @Test
+    @Issue("SECURITY-3252")
```

Comment:
wrong reference?

```suggestion
    @Issue("JENKINS-72611")
```

Reply:
Yup, updated in https://github.com/jenkinsci/credentials-plugin/pull/506/commits/eb2d6ca8db00a4d30ed74193fa6e6a73c14ed154